test/e2e/chaos/pod.go

```diff
@@ -214,3 +330,27 @@ func waitDeploymentReady(name, namespace string, cli kubernetes.Interface) error
 		return true, nil
 	})
 }
+
+func getUIDs(pods []corev1.Pod) []types.UID {
+	var ids []types.UID
+	for _, pod := range pods {
+		ids = append(ids, pod.UID)
+	}
+	return ids
+}
+
+func haveNewUID(newUIDs []types.UID, oldUIDs []types.UID) bool {
+	for _, newUID := range newUIDs {
+		found := false
+		for _, uid := range oldUIDs {
+			if uid == newUID {
+				found = true
+				break
+			}
+		}
+		if !found {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
The logic here seems complex. We can change them into `string` and sort the string array and compare.